docs/01-basics/00-basics.adoc

```diff
@@ -14,11 +14,12 @@
 
 // end::EN[]
 
+include::../what-to-expect-of-this-module.adoc[{include_configuration}]
+
 include::01-curriculum-structure-and-chronological-breakdown.adoc[{include_configuration}]
 
 include::02-timing-didactics-and-more.adoc[{include_configuration}]
 
-<<<
```

Comment:
...because I couldn't see any reason to have a page break within a chapter - especially if the sub chapters are so closely related, this additional whitespace didn't feel right but rather interrupted the visual flow.

Reply:
Well, it was a visually beautiful (read: adequate) break, but ok. 🙂 Your reasoning sounds reasonable.